lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/HandlerContainer.java

```diff
@@ -56,8 +56,13 @@
     this.name = name;
     this.codec = codec;
 
-    LOG.log(Level.FINER, "Instantiated HandlerContainer {0} with codec {1}",
-        new String[] {this.name, this.codec.getClass().getCanonicalName()});
+    LOG.log(Level.FINER, "Instantiated {0}", this.toString());
```

Comment:
Yes, please go ahead and remove the explicit `.toString()` call. Thanks for reviewing!